code/game/objects/items.dm

```diff
@@ -156,6 +156,35 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 	var/canMouseDown = FALSE
 
 
+	///Species Sprite Overrides
```

Comment:
ok so all of the item/clothing stuff can't be merged unless we know exactly what we're doing with it
we have tens of thousands of items per round and these are all wasted lists unless they're specifically needed, even with a typelist system like the rest of the code uses